app/helpers/edition_activity_buttons_helper.rb

```diff
@@ -11,8 +11,10 @@ def build_review_button(edition, activity, title)
 
   def review_buttons(edition)
     buttons = []
-    buttons << build_review_button(edition, "request_amendments", "Needs more work")
-    buttons << build_review_button(edition, "approve_review", "OK for publication")
+    if current_user.has_editor_permissions?(edition)
+      buttons << build_review_button(edition, "request_amendments", "Needs more work")
+      buttons << build_review_button(edition, "approve_review", "OK for publication")
+    end
```

Comment:
Personally, I think it would be better if the buttons were visible but disabled... but this is what was agreed, so happy to leave it as it is.

Reply:
I agree with you 😅